src/freedv_api.c

```diff
@@ -103,11 +103,13 @@ char *rx_sync_flags_to_text[] = {
 
 \*---------------------------------------------------------------------------*/
 
-struct freedv *freedv_open(int mode) {
-    return freedv_open_advanced(mode, NULL);
-}
+// Obsolete call
 
 struct freedv *freedv_open_advanced(int mode, struct freedv_advanced *adv) {
+    return freedv_open(mode);
+}
+
+struct freedv *freedv_open(int mode) {
```

Comment:
Nice @srsampson - you haven't changes the API, despite having an obsolete call!  :+1:  Removing _advanced would break many end user applications....